projects/RabbitMQ.Client/client/impl/BasicProperties.cs

```diff
@@ -109,13 +109,18 @@ public bool Persistent
         public abstract string ReplyTo { get; set; }
 
         /// <summary>
-        /// Convenience property; parses <see cref="ReplyTo"/> property using <see cref="PublicationAddress.Parse"/>,
+        /// Convenience property; parses <see cref="ReplyTo"/> property using <see cref="PublicationAddress.TryParse"/>,
         /// and serializes it using <see cref="PublicationAddress.ToString"/>.
-        /// Returns null if <see cref="ReplyTo"/> property cannot be parsed by <see cref="PublicationAddress.Parse"/>.
+        /// Returns null if <see cref="ReplyTo"/> property cannot be parsed by <see cref="PublicationAddress.TryParse"/>.
         /// </summary>
         public PublicationAddress ReplyToAddress
         {
-            get { return PublicationAddress.Parse(ReplyTo); }
+            get
+            {
+                PublicationAddress result;
+                PublicationAddress.TryParse(ReplyTo, out result);
```

Comment:
Could be simplified to:
```suggestion
                PublicationAddress.TryParse(ReplyTo, out var result);
```

And then the separation declaration on the line above is unneeded.